### PR TITLE
Add desktop support

### PR DESCRIPTION
**Pull request by e5l**

### CODE REVIEW

File: shared/src/jvmMain/kotlin/org/jetbrains/kotlinconf/utils/MobileLogger.jvm.kt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package org.jetbrains.kotlinconf.utils
+
+import io.ktor.client.plugins.logging.Logger
+
+actual fun mobileLogger(): Logger = object : Logger {
```
Comment (terrakok) [inline shared/src/jvmMain/kotlin/org/jetbrains/kotlinconf/utils/MobileLogger.jvm.kt:5]: appLogger?

File: shared/src/jvmMain/kotlin/org/jetbrains/kotlinconf/utils/Dispatchers.jvm.kt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package org.jetbrains.kotlinconf.utils
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+
+actual val Dispatchers.App: CoroutineDispatcher
```
Comment (terrakok) [inline shared/src/jvmMain/kotlin/org/jetbrains/kotlinconf/utils/Dispatchers.jvm.kt:6]: As I understand we may use Dispatchers.Main here

File: shared/src/jvmMain/kotlin/org/jetbrains/kotlinconf/storage/ApplicationStorage.jvm.kt [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            }
+
+            GlobalScope.launch(Dispatchers.IO) {
+                data.debounce(5_000)
+                    .collect { storageData ->
+                    val content = Json.encodeToString(storageData)
+                    runCatching {
+                        File("store.json").writeText(content)
+                    }
+                }
```
Comment (terrakok) [inline shared/src/jvmMain/kotlin/org/jetbrains/kotlinconf/storage/ApplicationStorage.jvm.kt:40]: I'd say 5 sec is too much. And code formatting is broken here.
Could you extract file to the property and use standard app directory on user's home?

**Review by terrakok — CHANGES_REQUESTED** [review]

File: shared/src/jvmMain/kotlin/org/jetbrains/kotlinconf/storage/ApplicationStorage.jvm.kt (hunk 3 above)
Comment (e5l) [reply shared/src/jvmMain/kotlin/org/jetbrains/kotlinconf/storage/ApplicationStorage.jvm.kt:40]: I'm unsure if using the user directory is a good idea, cause we provide no way to uninstall it.

File: shared/src/jvmMain/kotlin/org/jetbrains/kotlinconf/utils/MobileLogger.jvm.kt (hunk 1 above)
Comment (e5l) [reply shared/src/jvmMain/kotlin/org/jetbrains/kotlinconf/utils/MobileLogger.jvm.kt:5]: thanks, fixed

File: shared/src/jvmMain/kotlin/org/jetbrains/kotlinconf/utils/Dispatchers.jvm.kt (hunk 2 above)
Comment (e5l) [reply shared/src/jvmMain/kotlin/org/jetbrains/kotlinconf/utils/Dispatchers.jvm.kt:6]: The `Dispatchers.Main` purpose is UI updates. This dispatcher is used for IO operations

**Review by terrakok — APPROVED** [review]

**Pull request merged**